docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md

```diff
@@ -326,10 +326,10 @@ moment().format() → this
 
 ```js
 reportTaskDone (endTime, passed) {
-    const durationMs  = endTime - this.startTime;
-    const durationStr = this.moment.duration(durationMs).format('h[h] mm[m] ss[s]');
+    const durMs  = endTime - this.startTime;
+    const durStr = this.moment.duration(durMs).format('h[h] mm[m] ss[s]');
```

Comment:
Wouldn't it be better if we keep old names and just format it this way? @inikulin 
```
    const durationMs  = endTime - this.startTime;
    const durationStr = this.moment
                            .duration(durMs)
                            .format('h[h] mm[m] ss[s]');
```